tutorials/source-modeling/plot_beamformer_lcmv.py

```diff
@@ -0,0 +1,274 @@
+
+filters_vec = make_lcmv(evoked.info, forward, data_cov, reg=0.05,
+                        noise_cov=noise_cov, pick_ori='vector',
+                        weight_norm='unit-noise-gain', rank=None)
+
+###############################################################################
+# Apply the spatial filter
+# ------------------------
+# The spatial filter can be applied to different data types: raw, epochs,
+# evoked data or the data covariance matrix to gain a static image of power.
+# The function to apply the spatial filter to :class:`~mne.Evoked` data is
+# :func:`~mne.beamformer.apply_lcmv` which is
+# what we will use here. The other functions are
+# :func:`~mne.beamformer.apply_lcmv_raw`,
+# :func:`~mne.beamformer.apply_lcmv_epochs`, and
+# :func:`~mne.beamformer.apply_lcmv_cov`.
+
+stc = apply_lcmv(evoked, filters, max_ori_out='signed')
+stc_vec = apply_lcmv(evoked, filters_vec, max_ori_out='signed')
+
+###############################################################################
+# Visualize the reconstructed source activity
+# -------------------------------------------
+# We can visualize the source estimate in different ways, e.g. as an overlay
+# onto the MRI or as a glass brain.
+# The plots for the scalar beamformer show brain activity in the right temporal
+# lobe around 100 ms post stimulus. This is expected given the left-ear
+# auditory stimulation of the experiment.
+
+lims = [0.3, 0.45, 0.6]
+
+kwargs = dict(src=forward['src'], subject='sample', subjects_dir=subjects_dir,
+              initial_time=0.087, verbose=True)
+
+stc.plot(mode='stat_map', clim=dict(kind='value', pos_lims=lims), **kwargs)
+###############################################################################
+stc.plot(mode='glass_brain', clim=dict(kind='value', lims=lims), **kwargs)
+
+###############################################################################
+# Now let's visualize the vector beamformer case. Here we get three source time
+# courses out per voxel (one for each component of the dipole moment: x, y,
+# and z). To be able to visualize this, the plotting function combines those
+# estimates into one:
+
+stc_vec.plot(mode='stat_map', clim=dict(kind='value', pos_lims=lims), **kwargs)
+
+###############################################################################
+# Visualize the activity of the maximum voxel with all three components
+# ---------------------------------------------------------------------
+# We can also visualize all three components in the peak voxel. For this, we
+# will first find the peak voxel and then plot the time courses of this voxel.
+
+peak_vox, _ = stc_vec.get_peak(tmin=0.08, tmax=0.1, vert_as_index=True)
+
+ori_labels = ['x', 'y', 'z']
+fig, ax = plt.subplots(1)
+for ori, label in zip(stc_vec.data[peak_vox, :, :], ori_labels):
+    ax.plot(stc_vec.times, ori, label='%s component' % label)
+ax.legend(loc='lower right')
+ax.set(title='Activity per orientation in the peak voxel', xlabel='Time (ms)',
```

Comment:
```suggestion
ax.set(title='Activity per orientation in the peak voxel', xlabel='Time (s)',
```
time is in seconds here, not milliseconds